### PR TITLE
Minimize the CI to a minimum for now

### PR DESCRIPTION
**Pull request by nagisa**
The CI of Wasmer is doing a ton of stuff, and some of it depends on
various secrets and is otherwise not super necessary for our development
purposes.

Keep what we definitely need and delete everything else we don't derive
any use from right now.

### CODE REVIEW

**Comment by nagisa** [comment]
[CI](https://github.com/near/wasmer/runs/4298575000?check_suite_focus=true): 

```
Caused by:
  process didn't exit successfully: `/__w/wasmer/wasmer/target/release/deps/wasmer_middlewares-0758e0e2cac66b55` (signal: 11, SIGSEGV: invalid memory reference)
```

@nagisa: Uh-oh… *tries reproducing locally*
Local: All green!
@nagisa: Uh-oh…

**Comment by nagisa** [comment]
This is “done” to the point where this would be landable, but not to the point where I'm satisfied with the result. I was hoping that we would no longer call into `make` and could directly invoke `cargo test`/`cargo clippy`, but that route sadly causes additional errors to pop up.

I've no doubt that we'll continue looking into these issues as necessary, but for now this is probably good enough to unblock landing our other changes.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SHELL=/bin/bash
+SHELL=/usr/bin/env bash
```
Comment (matklad) [inline Makefile:1]: :heart:

**Review by matklad — APPROVED** [review]
bors r+

(Does this this work?)

**Pull request merged**